docs/source/javascripts/all.js

```diff
@@ -4,21 +4,47 @@ places({
   container: document.querySelector('#landing-demo')
 });
 
+// The following code is used to change the color of the navigation
+// depending the level of page scroll.
+const hero = document.querySelector('.hero-section');
```

Comment:
This seems like a merge issue, can you remove this block? since you have it already inside the next block